sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/impl/parser/BeamDDLTest.java

```diff
@@ -129,6 +129,25 @@ public void testParseCreateTable_withoutLocation() throws Exception {
         tableProvider.getTables().get("person"));
   }
 
+  @Test
+  public void testParseCreateTable_minimal() throws Exception {
+    TestTableProvider tableProvider = new TestTableProvider();
+    BeamSqlEnv env = BeamSqlEnv.withTableProvider(tableProvider);
+
+    env.executeDdl("CREATE TABLE person (\n" + "id INT) \n" + "TYPE text \n");
```

Comment:
multiline strings get mangled by spottlessApply

Reply:
Thanks for spotting that! Fixed.